cmd/plakar/subcommands/info/info.go

```diff
@@ -59,583 +46,57 @@ func cmd_info(ctx *appcontext.AppContext, repo *repository.Repository, args []st
-		fmt.Printf("Directory.MaxEntropy: %f\n", entry.Summary.Directory.MaxEntropy)
-		fmt.Printf("Directory.AvgEntropy: %f\n", entry.Summary.Directory.AvgEntropy)
-		fmt.Printf("Directory.HiEntropy: %d\n", entry.Summary.Directory.HiEntropy)
-		fmt.Printf("Directory.LoEntropy: %d\n", entry.Summary.Directory.LoEntropy)
-		fmt.Printf("Directory.MIMEAudio: %d\n", entry.Summary.Directory.MIMEAudio)
-		fmt.Printf("Directory.MIMEVideo: %d\n", entry.Summary.Directory.MIMEVideo)
-		fmt.Printf("Directory.MIMEImage: %d\n", entry.Summary.Directory.MIMEImage)
-		fmt.Printf("Directory.MIMEText: %d\n", entry.Summary.Directory.MIMEText)
-		fmt.Printf("Directory.MIMEApplication: %d\n", entry.Summary.Directory.MIMEApplication)
-		fmt.Printf("Directory.MIMEOther: %d\n", entry.Summary.Directory.MIMEOther)
-		fmt.Printf("Directory.Errors: %d\n", entry.Summary.Directory.Errors)
-		fmt.Printf("Directory.Children: %d\n", entry.Summary.Directory.Children)
-	}
-
-	iter, err := entry.Getdents(fs)
-	if err != nil {
-		return err
-	}
-	offset := 0
-	for child := range iter {
-		fmt.Printf("Child[%d].FileInfo.Name(): %s\n", offset, child.Stat().Name())
-		fmt.Printf("Child[%d].FileInfo.Size(): %d\n", offset, child.Stat().Size())
-		fmt.Printf("Child[%d].FileInfo.Mode(): %s\n", offset, child.Stat().Mode())
-		fmt.Printf("Child[%d].FileInfo.Dev(): %d\n", offset, child.Stat().Dev())
-		fmt.Printf("Child[%d].FileInfo.Ino(): %d\n", offset, child.Stat().Ino())
-		fmt.Printf("Child[%d].FileInfo.Uid(): %d\n", offset, child.Stat().Uid())
-		fmt.Printf("Child[%d].FileInfo.Gid(): %d\n", offset, child.Stat().Gid())
-		fmt.Printf("Child[%d].FileInfo.Username(): %s\n", offset, child.Stat().Username())
-		fmt.Printf("Child[%d].FileInfo.Groupname(): %s\n", offset, child.Stat().Groupname())
-		fmt.Printf("Child[%d].FileInfo.Nlink(): %d\n", offset, child.Stat().Nlink())
-		offset++
-	}
-
-	errors, err := snap1.Errors(pathname)
-	if err != nil {
-		return err
-	}
-	offset = 0
-	for err := range errors {
-		fmt.Printf("Error[%d]: %s: %s\n", offset, err.Name, err.Error)
-		offset++
-	}
-
-	return nil
-}
-
-func info_errors(repo *repository.Repository, snapshotID string) error {
-	prefix, pathname := utils.ParseSnapshotID(snapshotID)
-	if !strings.HasSuffix(pathname, "/") {
-		pathname = pathname + "/"
-	}
-
-	snap, err := utils.OpenSnapshotByPrefix(repo, prefix)
-	if err != nil {
-		return err
-	}
-	defer snap.Close()
-
-	errstream, err := snap.Errors(pathname)
-	if err != nil {
-		return err
-	}
-
-	for item := range errstream {
-		fmt.Printf("%s: %s\n", item.Name, item.Error)
+		return &InfoVFS{
+			RepositoryLocation: repo.Location(),
+			RepositorySecret:   ctx.GetSecret(),
+			SnapshotPath:       flags.Args()[1],
+		}, nil
 	}
 
-	return nil
+	fmt.Println("Invalid parameter. usage: info [snapshot|object|chunk|state|packfile|vfs]")
+	return nil, fmt.Errorf("Invalid parameter. usage: info [snapshot|object|chunk|state|packfile|vfs]")
```

Comment:
there is no `chunk` subcommand; probably another case of git merge resurrecting bits.  `errors` is also missing.